src/olympia/activity/utils.py

```diff
@@ -136,8 +138,21 @@ def log_and_notify(action, comments, note_creator, version):
         alog.user for alog in ActivityLog.objects.for_version(version)]
     # Collect add-on authors (excl. the person who sent the email.)
     addon_authors = set(version.addon.authors.all()) - {note_creator}
-    # Collect reviewers on the thread (again, excl. the email sender)
-    reviewer_recipients = set(log_users) - addon_authors - {note_creator}
+    # Collect staff that want a copy of the email
+    try:
+        cc_group = models.Group.objects.get(name='Activity Mail CC')
+        staff_cc = [user for user in cc_group.users.all()]
```

Comment:
Ultimately the query being made will be similar - MySQL should be smart enough to look at which groups match first, I think. I think I prefer the version with the `cc_group` variable.